### DISCUSSION TITLE
Double click to access/get candle information

### DISCUSSION THREAD

**Discussion by seyeint** [highfestiva/finplot#445] [Q&A] [answered]
Good afternoon, I'm working on a project related to filtering talent for a firm. 

Currently working on one functionality that would benefit from being able to simply double click a candle to retrieve its information for possible later usage - like choosing to execute a trade at a certain price - meaning it's low, high, open, close information. 
I would have no problem handling that information after retrieved, as I already know where to save it temporarily, so no need to explain anything besides the main point of _**double clicking as a function of retrieving candle information**_. 
Hopefully that's doable and sorry if it's an easy question.

Many thanks,
JP

**Comment by highfestiva** [comment, marked as answer]
For now use the badly named function `set_time_inspector(my_callback, ax=ax, when='double-click')`. It will call your function with x, y. See the S&P 500 example for more info.

**Reply by highfestiva** [reply]
Hereon that function more aptly named, see 7208f3fc5771caa3e366f7a8bc47a8df04a2a537 for more info.
